integration/e2e/metrics-generator/metrics_generator_test.go

```diff
@@ -0,0 +1,225 @@
+package serverless
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"math/rand"
+	"net/http"
+	"net/url"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/grafana/e2e"
+	thrift "github.com/jaegertracing/jaeger/thrift-gen/jaeger"
+	io_prometheus_client "github.com/prometheus/client_model/go"
+	"github.com/prometheus/common/expfmt"
+	"github.com/prometheus/prometheus/model/labels"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	util "github.com/grafana/tempo/integration"
+)
+
+const (
+	configMetricsGenerator = "config-metrics-generator.yaml"
+	prometheusImage        = "prom/prometheus:latest"
+)
+
+func TestMetricsGenerator(t *testing.T) {
```

Comment:
This e2e test is skipped with the current makefile targets, and the CI. We should add it there.

Reply:
Doh 🤦🏻